### ISSUE TITLE
Create params `MinSupply` and `MaxSupply`

### ISSUE THREAD

**Issue by lubtd** [tendermint/spn#493]
We currently prevent when send `MsgUpdateTotalSupply` removing an existing denomination from the `TotalSupply` of the campaign otherwise the coordinator has the ability to remove value from already minted vouchers. The existing denom can only be set to a new value or new denoms can be added in the total supply

However, the value of the supply for a denom has no range and can be set to `1`. We should create params to set of reasonable range for any use case for the minimum value and the maximum value of a total supply

**Comment by giunatale** [comment]
@lubtd don't we have the same issue when doing `MsgCreateCampaign`?

**Comment by lubtd** [comment]
Yes, you can provide a original `TotalSupply` when creating the campaign, therefore, it should be checked as well for this message

**Comment by giunatale** [comment]
I guess the other question I have is: should these params be on-chain and managed by governance or hard-coded?

I would prefer the first option but this would also require changing the keeper slightly (adding a `paramSpace`, e.g. from [cosmos-sdk](https://github.com/cosmos/cosmos-sdk/blob/master/x/distribution/keeper/keeper.go#L20))

**Comment by lubtd** [comment]
Yes, the name of the issue implicitly means we want to go with the `param` module

Unfortunately, we hadn't param automatic scaffolding when the module has been scaffolded, so this must be added manually as well
